egs/sre08/v1/sid/train_diag_ubm.sh

```diff
@@ -86,6 +85,7 @@ for f in $data/feats.scp $data/vad.scp; do
    [ ! -f $f ] && echo "$0: expecting file $f to exist" && exit 1
 done
 
+parallel_opts="-pe smp $num_threads"
```

Comment:
also, the "-pe smp N" format is not recommended now.  please use --num-threads N" instead.